src/nvim/version.c

```diff
@@ -1027,11 +1027,11 @@ void intro_message(int colon)
   };
 
   // blanklines = screen height - # message lines
-  blanklines = (int)Rows - (ARRAY_SIZE(lines) - 1);
+  blanklines = (int)Rows - (int)(ARRAY_SIZE(lines) - 1);
 
   // Don't overwrite a statusline.  Depends on 'cmdheight'.
   if (p_ls > 1) {
-    blanklines -= Rows - topframe->fr_height;
+    blanklines -= (int)Rows - (int)topframe->fr_height;
```

Comment:
`fr_height` looks to already be an int, unless I'm missing something. Perhaps this is for clarity?